src/System.IO.FileSystem/src/System/IO/UnixFileSystemObject.cs

```diff
@@ -150,9 +171,14 @@ public bool Exists
 
             public DateTimeOffset CreationTime
             {
-                // Not supported.
-                get { return default(DateTimeOffset); }
-                set { } // Not supported. Only some systems support "birthtime" from stat.
+                get
+                {
+                    EnsureStatInitialized();
+                    return (_fileinfo.flags & (uint)Interop.libcoreclr.FileInformationFlags.HasBTime) != 0 ?
+                        DateTimeOffset.FromUnixTimeSeconds(_fileinfo.btime) :
+                        default(DateTimeOffset);
+                }
+                set { } // Not supported
```

Comment:
This is following what the WinRT implementation does for such things, which is to ignore rather than throw. I'd be fine switching if that's what we decide is the right answer, but wet should do it for all such implementations... @ericstj?